ffmpeg+librav1e mingw-w64 build failure

Currently building librav1e with FFmpeg master branches results in 
```bash
gcc -D_FORTIFY_SOURCE=0 -D__USE_MINGW_ANSI_STDIO=1 -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -U__STRICT_ANSI__ -D__USE_MINGW_ANSI_STDIO=1 -D__printf__=__gnu_printf__ -D_POSIX_C_SOURCE=200112 -D_XOPEN_SOURCE=600 -DPIC -DOPJ_STATIC -mthreads -mtune=generic -O2 -pipe -fopenmp -DLIBTWOLAME_STATIC -DCACA_STATIC -DMODPLUG_STATIC -DCHROMAPRINT_NODLL -DZMQ_STATIC -DLIBXML_STATIC -DKVZ_STATIC_LIB -IC:/PROGRA~1/NVIDIA~2/CUDA/v10.2/include -std=c11 -fomit-frame-pointer -pthread -DLIBXML_STATIC -ID:/media-autobuild_suite/local64/include/harfbuzz -ID:/media-autobuild_suite/local64/include/libxml2 -ID:/media-autobuild_suite/local64/include/fribidi -ID:/media-autobuild_suite/local64/include/freetype2 -DLIBXML_STATIC -ID:/media-autobuild_suite/local64/include/libxml2 -ID:/media-autobuild_suite/local64/include/bs2b -DLIBXML_STATIC -ID:/media-autobuild_suite/local64/include/libxml2 -ID:/media-autobuild_suite/local64/include/freetype2 -ID:/media-autobuild_suite/local64/include/freetype2 -ID:/media-autobuild_suite/local64/include/fribidi -ID:/media-autobuild_suite/msys64/mingw64/include/libmodplug -ID:/media-autobuild_suite/msys64/mingw64/include/openjpeg-2.3 -ID:/media-autobuild_suite/local64/include/opus -ID:/media-autobuild_suite/local64/include/opus -ID:/media-autobuild_suite/local64/include/rav1e -LD:/media-autobuild_suite/local64/lib -c -o /tmp/ffconf.Z96vGm6X/test.o /tmp/ffconf.Z96vGm6X/test.c
In file included from D:/media-autobuild_suite/msys64/tmp/ffconf.Z96vGm6X/test.c:1:
D:/media-autobuild_suite/local64/include/rav1e/rav1e.h:375:9: error: unknown type name 'RaChromaSampling'
  375 | typedef RaChromaSampling RaChromaSampling;
      |         ^~~~~~~~~~~~~~~~
D:/media-autobuild_suite/msys64/tmp/ffconf.Z96vGm6X/test.c: In function 'check_rav1e_context_new':
D:/media-autobuild_suite/msys64/tmp/ffconf.Z96vGm6X/test.c:3:45: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
    3 | long check_rav1e_context_new(void) { return (long) rav1e_context_new; }
      |                                             ^
ERROR: rav1e >= 0.1.0 not found using pkg-config
```
rav1e.h
```c
...
/**
 * Chromaticity coordinates as defined by CIE 1931, expressed as 0.16
 * fixed-point values.
 */
typedef struct {
    /**
     * The X coordinate.
     */
    uint16_t x;
    /**
     * The Y coordinate.
     */
    uint16_t y;
} RaChromaticityPoint;

typedef RaChromaSampling RaChromaSampling;

/**
 * A rational number.
 */
typedef struct {
    /**
     * Numerator.
     */
    uint64_t num;
    /**
     * Denominator.
     */
    uint64_t den;
} RaRational;

...
```
I feel it may be related to https://github.com/xiph/rav1e/commit/41464e701e6ea2052120e1a14c7b9e68de3b563c

--- a/cbindgen.toml
+++ b/cbindgen.toml
@@ -9,7 +9,7 @@ cpp_compat = true
 
 [parse]
 parse_deps = true
-include = ['rav1e']
+include = ['rav1e', 'v_frame']
 
 [export]
 prefix = "Ra"

